client/src/modules/lead/EditLead/EditLeadContent/EditLeadTabs/AddActivity/LinkedTo/LeadAutocompleteWrapper/LeadAutocompleteWrapper.css

```diff
@@ -0,0 +1,43 @@
+.leadAutocomplete {
+  display: flex;
+  border: 1px solid #cbcccd;
+  align-content: center;
+  align-items: center;
+  flex-wrap: nowrap;
+  height: 32px;
+  border-radius: 3px;
+  margin-top: 8px;
+}
+
+.clearIcon {
+  width: 16px;
+  min-width: 16px;
```

Comment:
maybe you should set flex-grow/shrink 0 instead of setting min-width/height or even omit it completely